src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Figures {

```

Comment:
```suggestion
public enum Figure {

```

Reply:
I can't rename this enam, becouse I alredy have class "Figure"

Reply:
Fair enough, but a name in the plural isn't an option. Let's call it  `FigureType`